<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>labfeed</class>
 <widget class="QDialog" name="labfeed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Apparatus Registering</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>450</y>
     <width>160</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="pushButton_submit">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_exit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Specifications</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_specs">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>301</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_welcome">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Registering Apparatus in</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_setlab">
      <item>
       <property name="text">
        <string>Energy Lab</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Control systems Lab</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mechatronics Lab</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Aero Lab</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mechanics Lab</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Die and Mold</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Machine Shop</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Automotive lab</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Woodwork shop</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sheet metal shop</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Thermodynamics Lab</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ref and Air Lab</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CAD lab and drawing office</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>323</width>
     <height>363</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Apparatus ID: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_aid">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Apparatus Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Aname"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Condition</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_condition">
              <item>
               <property name="text">
                <string>Working</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Needs Consumables</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Needs Repair</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Beyond Repair</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Use</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_use">
            <item>
             <property name="text">
              <string>Manufacturing</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Testing</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Demonstration only</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Consumables</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_Consume">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Consume">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_description"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Quantity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_qty">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
